core/src/main/java/com/linecorp/armeria/client/retry/RetryingHttpClientBuilder.java

```diff
@@ -28,8 +30,12 @@
 public class RetryingHttpClientBuilder extends RetryingClientBuilder<
         RetryingHttpClientBuilder, RetryingHttpClient, HttpRequest, HttpResponse> {
 
+    private static final int DEFAULT_CONTENT_PREVIEW_LENGTH = 1024 * 1024; // 1 MiB
```

Comment:
Should we make this a flag as well? I'm not sure it's necessary though. It's up to you.